Documentation/k8s-pre-reqs.md

```diff
@@ -69,6 +69,61 @@ kubectl cluster-info
 
 Once you see a url response, your cluster is [ready for use by Rook](ceph-quickstart.md#deploy-rook).
 
+## Support for authenticated docker registries
+
+If you want to use an image from authenticated docker registry (e.g. for image cache/mirror), you'll need to 
+add an `imagePullSecret` to all relevant service accounts. This way all pods created by the operator (for service account: 
+`rook-ceph-system`) or all new pods in the namespace (for service account: `default`) will have the `imagePullSecret` added
```

Comment:
per other comment, perhaps ceph-specific examples could be in a subheading with the example below.